.github/workflows/pull_request.yml

```diff
@@ -250,6 +250,31 @@ jobs:
         with:
           token: ${{ secrets.CODECOV_TOKEN }}
           directory: ./pwa/coverage/
+  pwa-test-devbox:
+    name: "[pwa] Test w/ Devbox"
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v4
+      - name: Install devbox
+        uses: jetify-com/devbox-install-action@v0.13.0
+        with:
+          project-path: pwa
+          enable-cache: true
+      - name: Create dotenv
+        run: ./ops/pwa/create_dotenv.sh --env
+        env:
+          TBA_API_READ_KEY: TEST_KEY
+      - name: Install dependencies
+        run: devbox run -- npm ci
+        working-directory: pwa
+      - name: Run tests
+        run: devbox run -- npm test
+        working-directory: pwa
+      - name: Upload coverage to Codecov
```

Comment:
Not sure what happens if we double upload to codecov. I think we can just remove this?